terraform/modules/data-share-service/iam.tf

```diff
@@ -173,7 +173,12 @@ data "aws_iam_policy_document" "ecs_task_ssm_access" {
     actions = ["ssm:GetParameter"]
     resources = [
       aws_ssm_parameter.lev_api_client_name.arn,
-      aws_ssm_parameter.lev_api_client_user.arn
+      aws_ssm_parameter.lev_api_client_user.arn,
+      aws_ssm_parameter.prisoner_search_api_client_id.arn,
+      aws_ssm_parameter.prisoner_search_api_client_secret.arn,
+      aws_ssm_parameter.prisoner_event_queue_name.arn,
```

Comment:
I think this needs to go on the task execution role instead of the task role now. The task role is what is used at runtime, where as ECS assumes the execution role. We need ECS to fetch the values to inject into env vars

Reply:
ah ok got it